.lintr

```diff
@@ -4,3 +4,35 @@ linters: linters_with_defaults(
   object_usage_linter = NULL,
   object_name_linter = object_name_linter(styles = c("snake_case", "symbols"), regexes = c(ANL = "^ANL_?[0-9A-Z_]*$", ADaM = "^r?AD[A-Z]{2,3}_?[0-9]*$"))
   )
+exclusions: list(
+    "vignettes/teal-modules-general.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-association-plot.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-bivariate-plot.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-cross-table.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-data-table.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-outliers-module.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-regression-plots.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-response-plot.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-scatterplot-matrix.Rmd" = list(
+      object_name_linter = Inf
+    ),
+    "vignettes/using-scatterplot.Rmd" = list(
+      object_name_linter = Inf
+    )
+  )
```

Comment:
During the pkg cleanup we agreed that a common regex for `object_name_linter` should be the way to go.
So, we went with:
```
object_name_linter = object_name_linter(styles = c("snake_case", "symbols"), regexes = c(ANL = "^ANL_?[0-9A-Z_]*$", ADaM = "^r?AD[A-Z]{2,3}_?[0-9]*$"))
```
This solves our object name lint issue. Can we remove these file-level exclusions?

Reply:
Yes I think so. Let me do that

Reply:
Thanks! Also thanks for fixing warning with `color_setting`